benchmarks/imagenet/resnet50/ssley.py

```diff
@@ -0,0 +1,127 @@
+from typing import List, Tuple
+
+import torch
+from pytorch_lightning import LightningModule
+from torch import Tensor
+from torch.nn import Identity
+from torchvision.models import resnet50
+
+from lightly.loss.ssley_loss import SSLEYLoss
+from lightly.models.modules.heads import VICRegProjectionHead
+from lightly.models.utils import get_weight_decay_parameters
+from lightly.transforms.ssley_transform import VICRegTransform
```

Comment:
This import doesn't work as `ssley_transform` doesn't exist yet. I think it would make sense to add a `ssley_transform.py` module and create a `SSLEYTransform` which inherits from `VICRegTransform`. That way one doesn't have to remember that ssley uses the vicreg transform.